zppy/e3sm_diags.py

```diff
@@ -233,6 +233,10 @@ def add_climo_dependencies(
     depend_on_climo: Set[str] = set(
         [
             "lat_lon",
+            # Note: often `lat_lon_land` will require a different climo_subsection
+            # than the other sets (e.g., a climo subsection that includes land).
+            # That means this set will often need to be run as a separate subtask.
+            "lat_lon_land",
```

Comment:
Thanks for the PR. I was unsure about this line, and comment. But reading more of the code, I understand how dependencies handles differently for land vs atm, via "climate_section".